client/me/sidebar/index.jsx

```diff
@@ -1,68 +1,72 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	debug = require( 'debug' )( 'calypso:me:sidebar' );
+import React from 'react';
+import debugFactory from 'debug';
+import { connect } from 'react-redux';
+
+const debug = debugFactory( 'calypso:me:sidebar' );
 
 /**
  * Internal dependencies
  */
-var Sidebar = require( 'layout/sidebar' ),
+const Sidebar = require( 'layout/sidebar' ),
 	SidebarHeading = require( 'layout/sidebar/heading' ),
 	SidebarItem = require( 'layout/sidebar/item' ),
 	SidebarMenu = require( 'layout/sidebar/menu' ),
 	config = require( 'config' ),
-	layoutFocus = require( 'lib/layout-focus' ),
 	ProfileGravatar = require( 'me/profile-gravatar' ),
 	eventRecorder = require( 'me/event-recorder' ),
-	observe = require( 'lib/mixins/data-observe' ),
 	FormButton = require( 'components/forms/form-button' ),
 	userUtilities = require( 'lib/user/utils' );
 
-module.exports = React.createClass( {
+import { setNextLayoutFocus } from 'state/ui/layout-focus/actions';
+import { getCurrentUser } from 'state/current-user/selectors';
+
+const MeSidebar = React.createClass( {
 
 	displayName: 'MeSidebar',
```

Comment:
`displayName` becomes redundant when `React.createClass` is assigned to a named variable.